src/Console/InstallCommand.php

```diff
@@ -55,6 +64,34 @@ public function handle()
         return 1;
     }
 
+    /**
+     * Interact with the user to prompt them when the stack argument is missing.
+     *
+     * @param  \Symfony\Component\Console\Input\InputInterface  $input
+     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
+     * @return void
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        if ($this->argument('stack') === null && $this->option('inertia')) {
+            $input->setArgument('stack', 'vue');
+        }
```

Comment:
Do you think we can just remove the deprecated `--inertia` option instead of this?

Reply:
Think that will give errors for people who are using it in automated workflows. Might be best to wait for a major release to do that.